website/docs/tutorials/03-project-initialization.md

```diff
@@ -41,19 +41,35 @@ main = [
 ]
 
 ```
-Protostar's commands can be configured in the `protostar.toml`. This is number of arguments you would have to provide in the CLI otherwise. Protostar expects a section `["protostar.COMMAND_NAME"]` and a argument name with underscores (`_`) in place of dashes (`-`), for example:
+### Dynamic configuration
```

Comment:
I think Static and Dynamic are not very descriptive names and make no much sense in this context. What is dynamic about this argument? Are they changed by protostar dynamically?
Maybe `Project configuration` and `command arguments usign config` or something similar?

Reply:
Yep, not the best choice of words. It's dynamic from a Protostar developer perspective.